akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -499,7 +509,9 @@ object ShardCoordinator {
   private final case class RebalanceResult(shards: Set[ShardId])
 
   private[akka] object RebalanceWorker {
+
     final case class ShardRegionTerminated(region: ActorRef)
+
```

Comment:
spacy (not sure it's better)

Reply:
I don't remember to have done that. There are other formatting above. 

Anyway, will revert and build again to see what we get.